src/interpreter/map-validator.test.ts

```diff
@@ -1791,6 +1791,66 @@ describe('MapValidator', () => {
       });
     });
 
+    describe('result is enum', () => {
+      const profileAst = parseProfileFromSource(`usecase FaceDetection {
+        result {
+          emotions! emotions!
+        }!
+      }
+      
+      model emotions {
+        happiness! likelihood!
+        anger! likelihood!
+        sadness! likelihood!
+        surprise! likelihood!
+      }
+      
+      model likelihood enum {
+        unknown
+        veryUnlikely
+        unlikely
+        possible
+        likely
+        veryLikely
+      }`);
+
+      const validMapAst = parseMapFromSource(`map FaceDetection {
+        map result {
+          emotions: {
+            anger: "unknown",
+            happiness: "veryUnlikely",
```

Comment:
Can you add a case with backticks? I think that switches it from string literal to jessie script

Reply:
Oh, nice catch, I'll create new construct for that kind.

Reply:
Cool, using backticks fixes my problem in station 👍

Reply:
@lukas-valenta I got `NoSubstitutionTemplateLiteral` when debugging, so I added new construct for it, but I'm not sure if it will always have `text` property. Looking at [typescript types](https://github.com/microsoft/TypeScript/blob/main/src/compiler/types.ts), they have create functions always with text 🤷🏻‍♂️

Reply:
`NoSubstitutionTemplateLiteral extends TemplateLiteralLikeNode`

`TemplateLiteralLikeNode extends LiteralLikeNode`

```
export interface LiteralLikeNode extends Node {
    text: string;
    isUnterminated?: boolean;
    hasExtendedUnicodeEscape?: boolean;
}
```

`text` is mandatory, yes.